### PR TITLE
@W-9531390@: Replaced sanity test with a more robust and platform-agn…

### PR DESCRIPTION
**Pull request by jfeingold35**
…ostic script.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -141,16 +141,11 @@ jobs:
       - run: md test-results
 
       - run:
-          name: "List rules"
-          command: bin\run.cmd scanner:rule:list
+          name: "Unpackaged sanity test"
```
Comment (jbartolotta-sfdc) [inline .circleci/config.yml:144]: Can you add comments about the purpose of this run, same applies to `packaged`

File: .circleci/config.yml [hunk 2]
```diff
@@ -141,16 +141,11 @@ jobs:
...
+          command: smoke-tests\powershell.cmd bin\run.cmd
```
Comment (jbartolotta-sfdc) [inline .circleci/config.yml:145]: Is the "bin\run.cmd" only temporary until we create the tarballl? 

What is the advantage of running both "bin\run.cmd" and "sfdx" commands?

File: smoke-tests/SmokeTestGenerator.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
+/**
+ * This script auto-generates bash/cmd scripts meant to test smoke test the entire plugin. These scripts will do so by
+ * running a series of operations that collectively capture a vertical slice of the plugin, hitting every major piece
```
Comment (jbartolotta-sfdc) [inline smoke-tests/SmokeTestGenerator.js:3]: Can you add an example on how to invoke?

File: .circleci/config.yml (hunk 1 above)
Comment (jfeingold35) [reply .circleci/config.yml:144]: Will do.

File: .circleci/config.yml (hunk 2 above)
Comment (jfeingold35) [reply .circleci/config.yml:145]: Once we create the tarball, we can replace `bin\run.cmd` with the tarball invocation. But right now, `bin\run.cmd` is how we invoke the local build, and `sfdx` is how we invoke the installed version.

File: smoke-tests/SmokeTestGenerator.js (hunk 3 above)
Comment (jfeingold35) [reply smoke-tests/SmokeTestGenerator.js:3]: Will do.

**Review by jbartolotta-sfdc — APPROVED** [review]

**Pull request merged**